tools/accuracy_checker/openvino/tools/accuracy_checker/launcher/input_feeder.py

```diff
@@ -147,8 +149,16 @@ def prepare_scale_factor(image_meta):
                 return [[meta['scale_y'], meta['scale_x']] for meta in image_meta]
             return [[1, 1] for _ in image_meta]
 
-        meta_batch = extract_image_representations(data_representation_batch, meta_only=True)
+        data_batch, meta_batch = extract_image_representations(data_representation_batch, meta_only=False)
         image_infos = {}
+        if self.processed_image_info_inputs:
+            image_info_data = [data_batch[0].shape]
```

Comment:
```suggestion
            image_info_data = [np.shape(data) for data in data_batch]
```
your solution will work only for static shape and batch 1

Reply:
fixed